src/data/motebehov/useBehandleMotebehov.ts

```diff
@@ -32,7 +32,7 @@ export const useBehandleMotebehov = () => {
     },
     onSettled: () =>
       queryClient.invalidateQueries(motebehovQueryKey, {
-        refetchActive: false,
+        refetchType: "none",
```

Comment:
https://tanstack.com/query/v4/docs/react/guides/migrating-to-react-query-4#refetchactive--refetchinactive